scripts/qa_crowbarsetup.sh

```diff
@@ -449,6 +449,21 @@ function add_ha_repo()
     done
 }
 
+function add_ha12_repo()
+{
+    # TODO: add Updates-Test repo
+    local repo
+    for repo in SLE12-HA-{Pool,Updates}; do
+        if [ "$hacloud" == "2" -a "$repo" == "SLE12-HA-Updates-test" ] ; then
```

Comment:
If the lines were dropped, they would have to be added back in again once the `TODO` is done.